### ISSUE TITLE
PDU-MIB.txt url is now redirected, curls fails to follow

### ISSUE THREAD

**Issue by luizluca** [prometheus/snmp_exporter#649]
### Host operating system: output of `uname -a`
SLES15SP2

### snmp_exporter version: output of `snmp_exporter -version`
generator from 0.20.0

### What did you do that produced an error?
make mibs

### What did you expect to see?
It should work

### What did you see instead?
mibs/PDU-MIB.txt is a html page mentioning a redirect leading to 
```
level=warn ts=2021-04-23T17:00:03.321Z caller=tree.go:268 msg="Could not find node to override type" node=outletOperationalState
level=error ts=2021-04-23T17:00:03.322Z caller=main.go:130 msg="Error generating config netsnmp" err="cannot find oid '1.3.6.1.4.1.13742.4.1.2.2.1.3' to walk"
```

Adding "--location" to CURL_OPTS in Makefile fix the issue. I don't know if allowing curl to follow redirects might introduce a security issue.

Also, PDU.MIB location should be updated to avoid the redirect.

**Comment by luizluca** [comment]
The --location was removed with 633a459525708622e7b541922364413d2321b17b

**Comment by SuperQ** [comment]
Have you updated with https://github.com/prometheus/snmp_exporter/pull/647?

**Comment by luizluca** [comment]
No. I'm using the last released version 0.20.0.

Should '--location' return to avoid this kind of issue?

**Comment by SuperQ** [comment]
The fix is in master, you can copy the fix manually if you like.
